ScoutSuite/providers/aws/resources/elbv2/target_groups.py

```diff
@@ -0,0 +1,20 @@
+from ScoutSuite.providers.aws.facade.base import AWSFacade
+from ScoutSuite.providers.aws.resources.base import AWSResources
+
+
+class TargetGroups(AWSResources):
+    def __init__(self, facade: AWSFacade, region: str, load_balancer_arn: str):
+        super().__init__(facade)
+        self.region = region
+        self.load_balancer_arn = load_balancer_arn
+
+    async def fetch_all(self):
+        target_groups = await self.facade.elbv2.get_target_groups(self.region, self.load_balancer_arn)
+        for raw_target_group in target_groups:
+            arn, target_group = self._parse_target_groups(raw_target_group)
+            self[arn] = target_group
+
+    def _parse_target_groups(self, raw_target_group):
+            raw_target_group.pop('LoadBalancerArns')
+            arn = raw_target_group.pop('TargetGroupArn')
+            return arn, raw_target_group
```

Comment:
You should use https://github.com/nccgroup/ScoutSuite/blob/d5aba20188feb6838f79d00fbd58cf40bde5f140/ScoutSuite/providers/utils.py#L11 for the ID instead of the ARN.

Reply:
@x64-latacora Addressed the review comment